src/devices/tacxneo2/tacxneo2.cpp

```diff
@@ -900,8 +900,13 @@ void tacxneo2::errorService(QLowEnergyService::ServiceError err) {
 
 void tacxneo2::error(QLowEnergyController::Error err) {
     QMetaEnum metaEnum = QMetaEnum::fromType<QLowEnergyController::Error>();
-    emit debug(QStringLiteral("tacxneo2::error") + QString::fromLocal8Bit(metaEnum.valueToKey(err)) +
-               m_control->errorString());
+    auto key = metaEnum.valueToKey(err);
+    // auto keyString = QString::fromLocal8Bit(key); // this seg faults when key=="UnknownRemoteDeviceError"
```

Comment:
@cagnulein This might be of interest for the master branch as a potential problem with error reporting...on this branch for Trixter X-Dream in the TestDeviceDetectedValidNamesSettingsEnabled for the TaxNeo2 I was getting a seg fault in QString::fromLocal8Bit. The value coming out of metaEnum.valueToKey(err) was "UnknownRemoteDeviceError" - I'm not sure why yet. Also, it doesn't always fail.

Reply:
This is in QtCreator on Windows 11.